Setup/Product.wxs

```diff
@@ -64,6 +64,20 @@
           <Directory Id="ru_RU" Name="ru-RU" />
         </Directory>
       </Directory>
+      <Directory Id="LocalAppDataFolder">
+        <Directory Id="LocalAppDataOrganizationDir" Name="GitExtensions">
+          <Directory Id="LocalAppDataApplicationDir" Name="GitExtensions">
+            <Directory Id="UserPluginsDir" Name="UserPlugins">
+              <Component Id="UserPlugins" Guid="{81556E1B-270C-448D-A68D-FD30678D4827}">
+                <RemoveFolder Id="RemoveLocalAppDataOrganizationDir" Directory="LocalAppDataOrganizationDir" On="uninstall" />
+                <RemoveFolder Id="RemoveLocalAppDataApplicationDir" Directory="LocalAppDataApplicationDir" On="uninstall" />
+                <RemoveFolder Id="RemoveUserPluginsDir" Directory="UserPluginsDir" On="uninstall" />
+                <RemoveFolder Id="Removedir7351E76F113B364403061A39D2447F8D" Directory="dir7351E76F113B364403061A39D2447F8D" On="uninstall" />
```

Comment:
This directory is generated by `<HeatDirectory />` but is required to be listed for removal.

Reply:
Are there updates regarding this point, or is it still pending?

Reply:
I still don't know how to solve it.

Reply:
> All folders in user profile must be uninstalled, so I have defined their removal. But on of the folders is generated by <HeatDirectory /> build task and has generated id (Setup/Product.wxs#L75). Which may change unexpectedly..

> The reason why PM WIX component is generated is that its file structure may change in the future, so I tried to make the installer universal.

Top level directory created / deleted by installer should never change, PM should only operate under the top level. The installer can then recursively delete anything underneath during uninstall. Does the PM currently work this way?

Reply:
Yes, PM is placed under top level directory, but when I don't explicitly add `<RemoveFolder` for all PM stuff, compiler throws errors.

```
error LGHT0204: ICE64: The directory LocalAppDataOrganizationDir is in the user profile but is not listed in the RemoveFile table.
```

Reply:
Has become obsolete.